test/models/lookup_value_test.rb

```diff
@@ -121,6 +121,9 @@ def valid_attrs2
     env = FactoryGirl.create(:environment)
     pc = FactoryGirl.create(:puppetclass, :with_parameters, :environments => [env])
     key = pc.class_params.first
+    key.path += '\nfoo'
```

Comment:
1. `\nfoo` with single quotes wouldn't actually create a new line, is this really doing anything?
1. Would it be better to change the `foo` in the lookup_value below to `comment` or whatever's in the key path?